src/geneweaver/api/controller/genes.py

```diff
@@ -65,7 +66,7 @@ def get_gene_preferred(
         int, Path(format="int64", minimum=0, maxiumum=9223372036854775807)
     ],
     cursor: Optional[deps.Cursor] = Depends(deps.cursor),
-) -> dict:
+) -> GeneReturn:
```

Comment:
This should just be `-> Gene`.

Reply:
Related to comment above. In this case either a list or a single element is supported with the same model within the data element in the response